Template/HiP_MicroServiceTemplate.Model/Events/ICrudEvent.cs

```diff
@@ -23,17 +23,6 @@ public interface ICrudEvent : IEvent
         /// </summary>
         DateTimeOffset Timestamp { get; }
     }
-    public interface ICreateEvent : ICrudEvent
-    {
-    }
-
-    public interface IUpdateEvent : ICrudEvent
-    {
-    }
-
-    public interface IDeleteEvent : ICrudEvent
-    {
-    }
 
     public interface IUserActivityEvent : ICrudEvent
```

Comment:
Can't we remove the remaining two interfaces as well? They are not used anywhere.